<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hs_collect_setting</class>
 <widget class="QWidget" name="Hs_collect_setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="4">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>删除节点</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="node_name">
     <property name="text">
      <string>节点路径名</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>添加节点</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="node_input"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="node_names"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="chinese_name">
     <property name="text">
      <string>节点中文名</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="ch_input"/>
   </item>
   <item row="3" column="2">
    <widget class="QPlainTextEdit" name="chinese_names"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
